src/libraries/System.Diagnostics.StackTrace/tests/StackFrameTests.cs

```diff
@@ -37,10 +37,12 @@ public void Ctor_FNeedFileInfo(bool fNeedFileInfo)
 
         [Theory]
         [ActiveIssue("https://github.com/mono/mono/issues/15183", TestRuntimes.Mono)]
-        [InlineData(StackFrame.OFFSET_UNKNOWN)]
-        [InlineData(0)]
-        [InlineData(1)]
-        public void Ctor_SkipFrames(int skipFrames)
+        [InlineData(StackFrame.OFFSET_UNKNOWN, false)]
+        [InlineData(0, false)]
+        [InlineData(1, false)]
+        // This is highly dependent on reflection implementation and is not consistent across runtimes.
+        // The extra 'bool _' parameter avoids a potential reflection optimization for common signatures which may interfere with stack frame count.
```

Comment:
This will break again if this signature gets added to the list of optimized signatures. Should this be test be deleted, or ignore the potential extra frames instead?

Reply:
Correct - I would like to just remove this test but I was not sure on the long background\history here.

Reply:
I did remove that test. In general, the rest of the stack frame tests still verify the stack frame APIs work; this test verified constructors.

Also I will close the Mono https://github.com/mono/mono/issues/15183 issue once this PR is in.